tests/RequestTest.php

```diff
@@ -36,7 +36,7 @@ public function testGetBody()
     {
         $request = new Request($this->createNotification(), $production = false);
 
-        $this->assertEquals('{"aps":{}}', $request->getBody());
+        $this->assertEquals('{"aps":[]}', $request->getBody());
```

Comment:
I think "aps" should be an empty object but not an array. It is always an object.
It is not good if it will be an object in one case and an array in another case

Reply:
After looking more closely at the Apple documentation i agree with you. I found that you used an array in the `Payload::getDefaultPayloadStructure()` just changing this to an object and removing the `JSON_FORCE_OBJECT` was enough to fix the issue with the associative arrays and keep the tests happy :smile: